### ISSUE TITLE
Add environment variable to specify the release version

### ISSUE THREAD

**Issue by johanhaleby** [relaynetwork/lein-release#27]
I'd like to be able to specify the version to release either as an argument to the "release" task or as an environment variables similar to RELEASE_QUALIFIER instead of reading from project.clj. The reason for this is that we're releasing from Jenkins and we specify the release version as a part of the build. This version is also used for tagging Docker images.

**Comment by kyleburton** [comment]
Does RELEASE_VERSION work?  if this is present, it should be used, falling back to existing behavior.  Adding this would also imply adding CURRENT_SNAPSHOT_VERSION and NEXT_SNAPSHOT_VERSION for the "before", "release", and "after" versions.

**Comment by johanhaleby** [comment]
Yes that works, perhaps it should be documented a bit better though?

**Comment by johanhaleby** [comment]
Actually this doesn't work, I must have been confused when I wrote the previous comment

**Comment by kyleburton** [comment]
Sorry, I wasn't clear - I was offering a description of an enhancement.  Leiningen now comes with a release task - have you checked if that can do what you need?  Its possible to override its behavior in the project.clj file itself, including what steps get run and the arguments passed to them.   I'm no longer with Relay Network, I'm not able to update the repository or perform releases.  If the current maintainers aren't responsive and you still need the plugin, I can look at forking it and creating a new version under a different organization.

**Comment by johanhaleby** [comment]
Thanks for your comments. Didn't know about the release task in leiningen but after reading up on it it seems to be sufficient for my needs. Thanks again for the support!
